<ui version="4.0" >
 <class>ChooseColorDialog</class>
 <widget class="QDialog" name="ChooseColorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Assign Curve Color From File</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="colorGroup" >
     <property name="title" >
      <string>Assign Colors</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QFrame" name="colorFrame" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="vectorGroup" >
     <property name="title" >
      <string>Base Curve Color On</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_xVector" >
        <property name="text" >
         <string>&amp;X Vector</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="_yVector" >
        <property name="text" >
         <string>&amp;Y Vector</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>colorGroup</zorder>
  <zorder>_buttonBox</zorder>
  <zorder>vectorGroup</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>_xVector</tabstop>
  <tabstop>_yVector</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
